apps/db/test/player_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule DB.PlayerTest do
+  use DB.ModelCase
+
+  alias DB.{Game, Player, Repo, User}
+
+  @game_params %{
```

Comment:
might be a good idea to start pulling these out soon so that changes don't have to be applied in several files